### ISSUE TITLE
03-rtsp-server is NOK

### ISSUE THREAD

**Issue by natk16** [samtap/fang-hacks#342]
I'm having issues with the RTSP connecting. Have scoured the forums and various other blog sites but have not found any solutions yet. I followed the instructions as carefully as possible and got as far as setting everything up. I'll also preface that I am a complete newbie so please try to be as detailed as possible with any solutions you may have.

![image](https://user-images.githubusercontent.com/38484761/38935530-23448eea-4351-11e8-8802-0863da8b2a7a.png)

I read in another post that I need to run the '00-stop-cloud' service in order for RTSP to work. When I do that, it shows:

'Running script '00-stop-cloud'...
Stopping Cloud applications...
sh: you need to specify whom to kill
sh: you need to specify whom to kill
sh: you need to specify whom to kill'

I have tried several other SD cards. I've formatted them, re-applied the scripts and it's still showing up with the same errors. Am lost. Please help!

**Comment by natk16** [comment]
Just after I finished submitting the issue, I did some further digging and found that there was a similar problem by another user. I followed their [solution](https://github.com/samtap/fang-hacks/issues/239#issuecomment-344173093) and was able to resolve this. It also turns out that the copy of the fanghacks script I got was v1, not v2. This explains a lot. 

In addition to obtaining the most updated version, I reformatted my SD card to have a single partition (got rid of the exFat one) and flashed the v2 of the script using Etcher. Went through the whole process again, had to do a reboot along the way and voila, it works now!

**Comment by adrianmihalko** [comment]
Fanghacks V1, V2? What do you mean? I can't find any related info on releases page.

**Comment by natk16** [comment]
On https://github.com/samtap/fang-hacks/releases I scrolled down the page to the SD-Card image that was released on Jan, 16th 2017 (fanghacks_v0.0.1.zip). I should have used fanghacks_v0.2.0.zip, released on Jan, 26th 2017, shown at the top of the page. The instructions do not pertain to fanghacks_v0.0.1.zip, triggering my newbie confusion and hours of frustration. Just hacked a 2nd XiaoFang camera (with v2) and it took me 10-15mins tops. Good luck!
